.gitignore

```diff
@@ -4,3 +4,4 @@ target
 /cache/
 /rust.git/
 /rust/
+/results
```

Comment:
Running things like `profile_local` will populate a results directory which we'll never want to check in.